orttraining/orttraining/test/training_api/core/checkpoint_test.cc

```diff
@@ -470,4 +470,115 @@ TEST(CheckpointApiTest, LoadAndSaveNominalCheckpoint) {
     ASSERT_EQ(param->Data().Get<Tensor>().Shape().Size(), 1);
   }
 }
+
+/**
+ * Load ONNX model from file path, save into ORT checkpoint files,
+ * Then load it into ORT, compare with the initial parameter values.
+ */
+TEST(CheckpointApiTest, SaveOnnxModelAsCheckpoint_ThenLoad_WithExternalData) {
+  /// Phase 1 - Test Preparation
+  /// Prepare the data and dest folder for saving checkpoint.
+  /// Also cooked the data for test result comparison.
+
+  // Model path and trainable parameter name definitions.
+  auto model_uri = MODEL_FOLDER "transform/computation_reduction/gathernd/e2e.onnx";
+  std::vector<std::string> expected_trainable_param_names{
```

Comment:
I would make it a constexpr std::array<std::string_view>

Reply:
Marking as resolved so that the PR can go in before the code freeze, but will note for future follow-up PR